TODO.md

```diff
@@ -0,0 +1,16 @@
+# TODO
+
+## Now
+
+## Eventually
+
+- Get nox to work with downloaded pyodide/pyscript
+- Get rid of Poetry
```

Comment:
I'm up for listening to your POV. We can stick to Poetry. We can go back to just pip, perhaps with some pip extras. Or something else, as long as it doesn't put up a barrier for contribution.

Reply:
I think it would be good to have users create a single env for this, personally I enjoy using conda with poetry but that might be too confusing for someone just starting. Simple pip could work and it would lower the entry barrier since folks won't need to learn a new tool that they might not know about